src/library/LibraryWindow.vala

```diff
@@ -144,8 +144,8 @@ public class LibraryWindow : AppWindow {
 
     construct {
         set_default_size (
-            window_settings.get_int ("library-width"),
-            window_settings.get_int ("library-height")
+            window_settings.get_int ("library-position"),
+            window_settings.get_int ("library-size")
```

Comment:
These settings types are no longer "int".  You need to use the @get () function like this:

`window_settings.get ("window-size", "(ii)", out width, out height)`